libpod/runtime.go

```diff
@@ -88,12 +90,15 @@ type Runtime struct {
 	netPlugin       ocicni.CNIPlugin
 	ociRuntimePath  OCIRuntimePath
 	conmonPath      string
-	valid           bool
-	lock            sync.RWMutex
 	imageRuntime    *image.Runtime
 	firewallBackend firewall.FirewallBackend
 	lockManager     lock.Manager
 	configuredFrom  *runtimeConfiguredFrom
+
+	doRenumber bool
+
+	valid bool
+	lock  sync.RWMutex
```

Comment:
Can we add comments to the two booleans? The other fields are more or less self-explanatory (as the types can be looked up).